### DISCUSSION TITLE
Calling peaks and getting peak markers for clusters with few number of cells

### DISCUSSION THREAD

**Discussion by solvi808** [GreenleafLab/ArchR#2003] [Questions / Documentation]
Hello , 

I have some issue where I my smallest clusters with around 90 cells have very few number of peaks being called. My other clusters with 200-600 cells all have thousands of peaks called, however the clusters with few cells have only 40 marker peaks. This is an issue because I have less statistical power for motif enrichment analysis.

Are there some specific parameters or option I should be using so that peak calling is more sensitive for small clusters?

**Peak marker calling**
addGroupCoverages()
addReproduciblePeakSet()
addPeakMatrix()
 getMarkerFeatures()

I tried changing the parameter minCells from 25 to 20 in addGroupCoverages and addReproduciblePeakSet, it gave me a 20 or so more peaks for the small clusters, but thats a small improvement. And Im afraid to set this lower if there is some sort of QC limitation to setting it to say 15 or 10

**Comment by rcorces** [comment]
40 cells is just not enough to have meaningful data in such a sparse assay.

**Reply by solvi808** [reply]
The smallest clusters are 90 cells
